ckks/bootstrapping/dummy_bootstrapping.go

```diff
@@ -0,0 +1,49 @@
+package bootstrapping
+
+import (
+	"fmt"
+
+	"github.com/tuneinsight/lattigo/v4/ckks"
+	"github.com/tuneinsight/lattigo/v4/rlwe"
+)
+
+// DummyBootstrapper is a dummy bootstrapper that complies to
```

Comment:
Type name might be confusing:
- does not explain why it is "dummy"
- Could be useful outside testing

Reply:
done